// @IntusFacultas/design-system/theme.js
var NIWSTheme = {
  TASK: {
    color: {
      color: "#444444",
      hover: "#FFFFFF",
      focus: "#444444"
    },
    background: {
      color: "#CBE6F7",
      hover: "#51BAF4",
      focus: "#CBE6F7"
    },
    border: {
      color: "2px solid #CBE6F7",
      hover: "2px solid #51BAF4",
      focus: "2px solid #51BAF4"
    }
  },
  REWORK: {
    color: {
      color: "#444444",
      hover: "#FFFFFF",
      focus: "#444444"
    },
    background: {
      color: "#FFC364",
      hover: "#ED9406",
      focus: "#FFC364"
    },
    border: {
      color: "2px solid #FFC364",
      hover: "2px solid #ED9406",
      focus: "2px solid #ED9406"
    }
  },
  START: {
    color: {
      color: "#444444",
      hover: "#FFFFFF",
      focus: "#444444"
    },
    background: {
      color: "#B7F7DC",
      hover: "#2EE591",
      focus: "#B7F7DC"
    },
    border: {
      color: "2px solid #B7F7DC",
      hover: "2px solid #2EE591",
      focus: "2px solid #2EE591"
    }
  },
  COMPLETE: {
    color: {
      color: "#444444",
      hover: "#FFFFFF",
      focus: "#444444"
    },
    background: {
      color: "#E0CEF4",
      hover: "#735D87",
      focus: "#E0CEF4"
    },
    border: {
      color: "2px solid #E0CEF4",
      hover: "2px solid #735D87",
      focus: "2px solid #735D87"
    }
  },
  CANCEL: {
    color: {
      color: "#444444",
      hover: "#FFFFFF",
      focus: "#444444"
    },
    background: {
      color: "#DDA8A8",
      hover: "#964545",
      focus: "#DDA8A8"
    },
    border: {
      color: "2px solid #DDA8A8",
      hover: "2px solid #964545",
      focus: "2px solid #964545"
    }
  }
};
var TextTheme = {
  Normal: { color: "#444" },
  Dark: { color: "#e0e0e0" },
  PortionMarkingDark: { color: "#cbd5f4" },
  LightBlue: { color: "#41BEE8" },
  AirBlue: { color: "#2999F4" },
  MissileOrange: { color: "#FF7200" },
  CyberYellow: { color: "#F7C82C" },
  SpaceTeal: { color: "#0AD1C3" },
  INTsMagenta: { color: "#FC49C9" },
  RegionGreen: { color: "#C8E552" },
  Sky: { color: "#80CCFF" },
  Apricot: { color: "#FCB681" },
  Aqua: { color: "#77FCD5" },
  Sunny: { color: "#F9EC75" },
  Moss: { color: "#DFFF92" },
  Pink: { color: "#F2B4F9" },
  Deep: { color: "#002339" },
  Bark: { color: "#481F0C" },
  Jungle: { color: "#015149" },
  Spice: { color: "#705405" },
  Olive: { color: "#4C5612" },
  Plum: { color: "#361329" }
};
var AlertTheme = {
  warning: {
    color: {
      color: "#222",
      hover: "#222",
      focus: "#222"
    },
    background: {
      color: "#fec700",
      hover: "#fec700",
      focus: "#fec700"
    },
    border: {
      color: "#fec700",
      hover: "#fec700",
      focus: "#fec700"
    }
  },
  danger: {
    color: {
      color: "#fff",
      hover: "#fff",
      focus: "#fff"
    },
    background: {
      color: "#890620",
      hover: "#890620",
      focus: "#890620"
    },
    border: {
      color: "#670014",
      hover: "#670014",
      focus: "#670014"
    }
  },
  success: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#0B7C40",
      hover: "#00642E",
      focus: "#0B7C40"
    },
    border: {
      color: "#00642E",
      hover: "#00642E",
      focus: "#00642E"
    }
  },
  info: {
    color: {
      color: "#000",
      hover: "#000",
      focus: "#000"
    },
    background: {
      color: "#4E8098",
      hover: "#4E8098",
      focus: "#4E8098"
    },
    border: {
      color: "#316780",
      hover: "#316780",
      focus: "#316780"
    }
  }
};
var Theme = {
  Light: {
    color: {
      color: "#222",
      hover: "#222",
      focus: "#222"
    },
    background: {
      color: "#f8f9fa",
      hover: "#DDE4E9",
      focus: "#f8f9fa"
    },
    border: {
      color: "#DDE4E9",
      hover: "#DDE4E9",
      focus: "#DDE4E9"
    }
  },
  Secondary: {
    color: {
      color: "#fff",
      hover: "#fff",
      focus: "#fff"
    },
    background: {
      color: "#6c757d",
      hover: "#525D67",
      focus: "#6c757d"
    },
    border: {
      color: "#525D67",
      hover: "#525D67",
      focus: "#525D67"
    }
  },
  Dark: {
    color: {
      color: "#fff",
      hover: "#fff",
      focus: "#fff"
    },
    background: {
      color: "#343a40",
      hover: "#23272b",
      focus: "#343a40"
    },
    border: {
      color: "#4F575E",
      hover: "#4F575E",
      focus: "#4F575E"
    }
  },
  Primary: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#4357AD",
      hover: "#2940A1",
      focus: "#4357AD"
    },
    border: {
      color: "#2940A1",
      hover: "#2940A1",
      focus: "#2940A1"
    }
  },
  Info: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#58B0AE",
      hover: "#36938F",
      focus: "#58B0AE"
    },
    border: {
      color: "#36938F",
      hover: "#36938F",
      focus: "#36938F"
    }
  },
  Warning: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#ED7D3A",
      hover: "#D15A14",
      focus: "#ED7D3A"
    },
    border: {
      color: "#D15A14",
      hover: "#D15A14",
      focus: "#D15A14"
    }
  },
  Danger: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#7C0002",
      hover: "#560002",
      focus: "#7C0002"
    },
    border: {
      color: "#560002",
      hover: "#560002",
      focus: "#560002"
    }
  },
  Success: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#0B7C40",
      hover: "#00642E",
      focus: "#0B7C40"
    },
    border: {
      color: "#00642E",
      hover: "#00642E",
      focus: "#00642E"
    }
  },
  LightBlue: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#41BEE8",
      hover: "#38a5ca",
      focus: "#38a5ca"
    },
    border: {
      color: "#38a5ca",
      hover: "#38a5ca",
      focus: "#38a5ca"
    }
  },
  AirBlue: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#2999F4",
      hover: "#2386d7",
      focus: "#2386d7"
    },
    border: {
      color: "#2386d7",
      hover: "#2386d7",
      focus: "#2386d7"
    }
  },
  MissileOrange: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#FF7200",
      hover: "#C65800",
      focus: "#FF7200"
    },
    border: {
      color: "#C65800",
      hover: "#C65800",
      focus: "#C65800"
    }
  },
  CyberYellow: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#F7C82C",
      hover: "#D4A609",
      focus: "#F7C82C"
    },
    border: {
      color: "#D4A609",
      hover: "#D4A609",
      focus: "#D4A609"
    }
  },
  SpaceTeal: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#0AD1C3",
      hover: "#00BBAE",
      focus: "#0AD1C3"
    },
    border: {
      color: "#00BBAE",
      hover: "#00BBAE",
      focus: "#00BBAE"
    }
  },
  INTsMagenta: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#FC49C9",
      hover: "#FA1FBD",
      focus: "#FC49C9"
    },
    border: {
      color: "#FA1FBD",
      hover: "#FA1FBD",
      focus: "#FA1FBD"
    }
  },
  RegionGreen: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#C8E552",
      hover: "#B0CF2E",
      focus: "#C8E552"
    },
    border: {
      color: "#B0CF2E",
      hover: "#B0CF2E",
      focus: "#B0CF2E"
    }
  },
  Sky: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#80CCFF",
      hover: "#57BDFF",
      focus: "#80CCFF"
    },
    border: {
      color: "#57BDFF",
      hover: "#57BDFF",
      focus: "#57BDFF"
    }
  },
  Apricot: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#FCB681",
      hover: "#D88B51",
      focus: "#FCB681"
    },
    border: {
      color: "#D88B51",
      hover: "#D88B51",
      focus: "#D88B51"
    }
  },
  Aqua: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#77FCD5",
      hover: "#25EEB1",
      focus: "#77FCD5"
    },
    border: {
      color: "#25EEB1",
      hover: "#25EEB1",
      focus: "#25EEB1"
    }
  },
  Sunny: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#F9EC75",
      hover: "#D7C949",
      focus: "#F9EC75"
    },
    border: {
      color: "#D7C949",
      hover: "#D7C949",
      focus: "#D7C949"
    }
  },
  Moss: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#DFFF92",
      hover: "#C5FF38",
      focus: "#DFFF92"
    },
    border: {
      color: "#C5FF38",
      hover: "#C5FF38",
      focus: "#C5FF38"
    }
  },
  Pink: {
    color: {
      color: "#222",
      focus: "#222",
      hover: "#222"
    },
    background: {
      color: "#F2B4F9",
      hover: "#E486EE",
      focus: "#F2B4F9"
    },
    border: {
      color: "#E486EE",
      hover: "#E486EE",
      focus: "#E486EE"
    }
  },
  Deep: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#002339",
      hover: "#000B12",
      focus: "#002339"
    },
    border: {
      color: "#000B12",
      hover: "#000B12",
      focus: "#000B12"
    }
  },
  Bark: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#481F0C",
      hover: "#2B0E01",
      focus: "#481F0C"
    },
    border: {
      color: "#2B0E01",
      hover: "#2B0E01",
      focus: "#2B0E01"
    }
  },
  Jungle: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#015149",
      hover: "#003933",
      focus: "#015149"
    },
    border: {
      color: "#003933",
      hover: "#003933",
      focus: "#003933"
    }
  },
  Spice: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#705405",
      hover: "#4A3700",
      focus: "#705405"
    },
    border: {
      color: "#4A3700",
      hover: "#4A3700",
      focus: "#4A3700"
    }
  },
  Olive: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#4C5612",
      hover: "#2F3703",
      focus: "#4C5612"
    },
    border: {
      color: "#2F3703",
      hover: "#2F3703",
      focus: "#2F3703"
    }
  },
  Plum: {
    color: {
      color: "#fff",
      focus: "#fff",
      hover: "#fff"
    },
    background: {
      color: "#361329",
      hover: "#26081B",
      focus: "#361329"
    },
    border: {
      color: "#26081B",
      hover: "#26081B",
      focus: "#26081B"
    }
  },
  ...NIWSTheme,
  ...AlertTheme
};

export default Theme;
export { NIWSTheme, TextTheme, AlertTheme, Theme };
